packages/app/app/components/PlayQueue/index.js

```diff
@@ -54,7 +54,7 @@ class PlayQueue extends React.Component {
 
     return this.props.items.map((el, i) => {
       return (
-        <Draggable key={el.uuid} index={i} draggableId={el.uuid}>
+        <Draggable key={`${el.uuid}+${i}`} index={i} draggableId={`${el.uuid}+${i}`}>
```

Comment:
this isn't any better than just using the index as the key, which i know is bad, but I want to leave it in to remind myself to find better ways to create unique keys. will open up a new issue soon.